src/status_im/multiaccounts/recover/core.cljs

```diff
@@ -222,8 +222,8 @@
   [{:keys [db] :as cofx}]
   (let [storage-type (get-in db [:intro-wizard :selected-storage-type])]
     (if (= storage-type :advanced)
-      {:dispatch [:recovery.ui/keycard-option-pressed]})
-    (navigation/navigate-to-cofx cofx :recover-multiaccount-enter-password nil)))
+      {:dispatch [:recovery.ui/keycard-option-pressed]}
```

Comment:
why does it need to be dispatched? Can't it just call the `recovery-keycard-selected` fn directly?

Reply:
@yenda There is a circular dependency issue. `hardwallet.core` depends on `recover.core`.

Reply:
can you please create an issue and add a TODO here with the issue number saying that the circular dep needs to be fixed and this dispatch needs to be removed and assign an bounty M